hw03_samundra.py

```diff
@@ -0,0 +1,110 @@
+#name:Samundra Adhikari
+#username: adhikaris2
+#googledocs: https://docs.google.com/document/d/1Mj5nshMpoanpbSKld58QdK__H9CcjM_WI2m9WeCyrzo/edit#heading=h.66gmzdtk59q2
+# Drawing a cat.
+#Acknowlege: Used ChatGPT to ask questions, and used textbook.
```

Comment:
Use the header block formatting provided in the template.

Reply:
I have done the formatting as per the template. Thank you!